main/guides/coreeval/proposal.md

```diff
@@ -0,0 +1,59 @@
+# Code the Proposal
+
+You will need to write a proposal script that requests storageNode powers and passes them to the contract via privateArgs. For example, [here](https://github.com/Agoric/agoric-sdk/blob/master/packages/inter-protocol/test/psm/gov-add-psm.js) is the proposal we created for the PSM contract:
```

Comment:
_question_:
The script below doesn't make use of `storageNode`. What is it meant to highlight or explain?

Reply:
I rewrote the text blurbs at the top of both this page & the README.md page. Take a look at both, please.